_IBC/composablepolkadot-picasso.json

```diff
@@ -0,0 +1,31 @@
+{
+    "$schema": "../ibc_data.schema.json",
+    "chain_1": {
+      "chain_name": "composablepolkadot",
+      "client_id": "10-grandpa-28",
+      "connection_id": "connection-23"
+    },
+    "chain_2": {
+      "chain_name": "picasso",
+      "client_id": "10-grandpa-28",
+      "connection_id": "connection-23"
+    },
+    "channels": [
+      {
+        "chain_1": {
+          "channel_id": "channel-15",
+          "port_id": "transfer"
+        },
+        "chain_2": {
+          "channel_id": "channel-15",
```

Comment:
it seems very unlikely that both sides have the same client-id, connection-id, and channel-id; usually these are all different between chain 1 and chain 2

Reply:
checked with our bridging dev. He mentioned that it was symmetric on purpose as that's what they started with connecting to two parachains with IBC. And didn't want to change it afterwards given it works

Reply:
Thanks for confirming